infrastructure-playbooks/purge-cluster.yml

```diff
@@ -303,6 +303,11 @@
     failed_when: false
     register: ceph_lockbox_partition_to_erase_path
 
+  - name: see if ceph-volume is installed
+    shell: "which ceph-volume"
```

Comment:
why not using `command` instead? Also using `command -v` is generally preferred.

Reply:
@andrewschoen I think @leseb is means 
```
- name: see if ceph-volume is installed
  command: command -v ceph-volume
```

eg (in a shell) :

```
# command -v ceph-volume
/sbin/ceph-volume
#
```

Reply:
I'm still curious why `command -v` would be better than `which ceph-volume` though

Reply:
https://github.com/koalaman/shellcheck/wiki/SC2230

@andrewschoen please use `command -v ceph-volume`. Thanks

Reply:
@leseb ok, I've made that change.